frontend/src/components/Registration.tsx

```diff
@@ -8,18 +10,27 @@ const layout = {
 const tailLayout = {
     wrapperCol: { offset: 4, span: 16 },
 };
+class Registration extends React.Component{
 
-const Registration = () => {
-    const onFinish = (values: any) => {
-        console.log('Success:', values);
+    onFinish = async (values: any) => {
+        axios.post("http://localhost:3000/users", values)
```

Comment:
Use `async await` instead of promises.

Reply:
You are still using `.then` and `.catch`.

Use `try catch` statement instead and you can access `res` variable without `.then` like so:

```
const res = await axios.post("http://localhost:3000/users", values)
```